front/app/components/UI/MentionsTextArea/index.tsx

```diff
@@ -78,6 +79,9 @@ export interface Props {
   ariaLabel?: string;
   children?: React.ReactNode;
   roles?: MentionRoles[];
+  trigger?: string;
+  // Needs explanation
+  idAttribute?: 'slug' | 'id';
```

Comment:
@alexander-cit Was it necessary to introduce the 'id' part? It seems to work fine with slug in the initiativeForm. If not needed, I'll remove this again.

Reply:
AFAIR this component gives an array of objects `{ display: '', slug: '' }` by default. And `idAttribute` makes it `{ display: '', id: '' }`. But if you made it work without it, it's great!

Reply:
Oh, now I get it. Then your code is needed. Let me see if I can make it slightly clearer.